README.md

```diff
@@ -6,24 +6,20 @@ Privacy-focused mapping application for Android
 
 ## API Keys
 1. Go to https://mapzen.com/developers/ and auth with Github
-2. Create Mapzen Search, Turn By Turn, and Vector Tiles keys
+2. Create a Mapzen API key
 3. Add keys to `~/.gradle/gradle.properties` or use as command line arguments
```

Comment:
Change "keys" -> "key" and "arguments" -> "argument" since now it is a single key.